res/css/structures/auth/_Login.scss

```diff
@@ -88,13 +88,12 @@ limitations under the License.
 div.mx_AccessibleButton_kind_link.mx_Login_forgot {
     display: block;
     margin: 0 auto;
```

Comment:
These will be able to be removed by creating a wrapper and setting `kind="link_inline"` in favor of `link`.